filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -219,7 +219,7 @@ DECL_DRIVER_API_R_N(backend::TextureHandle, importTexture,
         backend::TextureUsage, usage)
 
 DECL_DRIVER_API_R_N(backend::SamplerGroupHandle, createSamplerGroup,
-        uint32_t, size)
+        uint32_t, size, const char*, name)
```

Comment:
Using a raw pointer here is dangerous. I'm *okay* with it if we document/make sure that on the caller side we use string literals. And I think it's the right thing to do. I don't want to add heap allocations for a debug feature like this.

Given that, we don't need to use a `CString` on the backend side (which will heap allocate). Just store the raw pointer there too.